cabal-install/tests/IntegrationTests2.hs

```diff
@@ -0,0 +1,258 @@
+module Main where
+
+import Distribution.Client.DistDirLayout
```

Comment:
It doesn't depend on all the cabal-install modules, but yes fair point. Once we feel we can rely on the named internal libs feature in Cabal then we could use it here.